src/main/java/org/firstPF/controllers/CustomerController.java

```diff
@@ -1,15 +1,69 @@
 package org.firstPF.controllers;
 
+import org.firstPF.entities.Customer;
 import org.firstPF.services.CustomerService;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.Optional;
 
 @RestController
+@RequestMapping("/customers")
 public class CustomerController {
     private final CustomerService customerService;
 
     @Autowired
     public CustomerController(CustomerService customerService) {
         this.customerService = customerService;
     }
+
+    @PostMapping
+    public ResponseEntity<Customer> createCustomer(@RequestBody Customer customer) {
+        Customer createdCustomer = customerService.createCustomer(customer);
+        return new ResponseEntity<>(createdCustomer, HttpStatus.CREATED);
```

Comment:
Doporučuju spíše používat ResponseEntity.status(HttpStatus.CREATED).body(createdCustomer). Je to přehlednější